tests/support/expectations.js

```diff
@@ -20,10 +22,11 @@ module.exports = function (request, config) {
       };
 
       var oldSet = request.Request.prototype.set;
-      request.Request.prototype.set = function (values) {
-        headers = values;
-
-        return oldSet.call(this, values);
+      request.Request.prototype.set = function (field, value) {
+        if (typeof field === 'object') { // spy on set to collect the used arguments
+          headers = field;
+        }
+        return oldSet.call(this, field, value);
```

Comment:
It's not defined on our coding style, but could you put an empty line before `return` statement ? 👼